onramp_compassion/mappings/correspondence_mapping.py

```diff
@@ -146,3 +136,22 @@ def _process_odoo_data(self, odoo_data):
             del odoo_data['correspondant_id']
             if sponsorship:
                 odoo_data['sponsorship_id'] = sponsorship.id
+        # Replace dict by a tuple for the ORM update/create
+        if 'page_ids' in odoo_data:
+            for page in odoo_data['page_ids']:
+                odoo_data['page_ids'].remove(page)

```

Comment:
It is a bad practice to modify the list being accessed by the for loop, and can end in unexpected behaviours. I would rather construct the converted value in a separate variable and at the end of the process re-assign the value of the field (`odoo_data['page_ids'] = new_value`)